### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by keshav-space** [keshav-space/safenotes#86] [Announcements]
## What's Changed [TL;DR]
* Auto-scroll for long notes in text-editor
* Auto-scroll for auth pages
* Support for Chines, French, and Portuguese
* New backup workflow - migrate to new privacy-friendly API

## What's Changed [Details]
* Support  multiple languages by @keshav-space in https://github.com/keshav-space/safenotes/pull/67
* Support Simplified Chinese by @ghoevil in https://github.com/keshav-space/safenotes/pull/71
* New CoC by @comradekingu in https://github.com/keshav-space/safenotes/pull/70
* Translations update from Hosted Weblate by @weblate in https://github.com/keshav-space/safenotes/pull/69
* Translations update from Hosted Weblate by @weblate in https://github.com/keshav-space/safenotes/pull/74
* Full-description in Chinese is done by @ghoevil in https://github.com/keshav-space/safenotes/pull/75
* Translations update from Hosted Weblate by @weblate in https://github.com/keshav-space/safenotes/pull/76
* Translations update from Hosted Weblate by @weblate in https://github.com/keshav-space/safenotes/pull/78
* Translations update from Hosted Weblate by @weblate in https://github.com/keshav-space/safenotes/pull/79
* Auto-scroll long text in text-editor by @keshav-space in https://github.com/keshav-space/safenotes/pull/82
* Translations update from Hosted Weblate by @weblate in https://github.com/keshav-space/safenotes/pull/83

## New Contributors
* @ghoevil made their first contribution in https://github.com/keshav-space/safenotes/pull/71
* @comradekingu made their first contribution in https://github.com/keshav-space/safenotes/pull/70
* @weblate made their first contribution in https://github.com/keshav-space/safenotes/pull/69

**Full Changelog**: https://github.com/keshav-space/safenotes/compare/v2.0.1...v2.1.0

`Note: If not sure of your device architecture use safenotes-v2.1.0-all.apk`

<hr /><em>This discussion was created from the release <a href='https://github.com/keshav-space/safenotes/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by comradekingu** [comment]
New non-CoC*. No conduct or shift in power codified or pledged allegiance to. :)
